docs/rules/periphery-rules.rst

```diff
@@ -321,7 +321,7 @@
      - 
 
 
-.. figure:: periphery/p018-x_dotdash.svg
+.. figure:: periphery\p018-x_dotdash.svg
```

Comment:
I imagine it's just because I'm on a Windows10 machine here (i.e. `os.path` doesn't yield the same output) , and I'm generating this manually rather than through a CICD pipeline. I can try again through WSL though I'll need to rebuild the `virtualenv` in there.

Reply:
Ran this on a Mac OSX machine and was able to fix this.